src/scenes/CreateEvent/store.js

```diff
@@ -137,13 +143,15 @@ const createEvent = async ({
 }) => {
   try {
     // Construct params
-    const paramsHex = web3.eth.abi.encodeParameters(
+    const paramsHex = web3_eth_abi.encodeParameters(
```

Comment:
move this function to the TransactionStore. IIRC, the Component stores were just to handle specific UI vars that needed to persist. TransactionStore is more like a bg biz logic type of store.

Reply:
duplicate func? its already in the TransactionStore.

Reply:
yeah. I just started at this place to verify if the function works, I have 5 buttons at empty placeholder page to click. will move all the contract method to the transaction store later. Right now just let you see if the function in transaction store is okay, so i can continue to move other functions.

Reply:
Gotcha. Ya the encoding with 1.0 and execution of the tx with older web3 looks correct.